security/s2a/internal/crypter/aesgcm_test.go

```diff
@@ -0,0 +1,513 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package crypter
+
+import (
+	"bytes"
+	"fmt"
+	. "google.golang.org/grpc/security/s2a/internal/crypter/testutil"
```

Comment:
Got it, I was debating on using a dot import here or not and I initially decided to use it since otherwise we'll have to write `testutil` all across the test. I updated it to use the fully qualified name now.

Reply:
SG, thanks!